### PR TITLE
[TDV-691] Feature/offline app details

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -1019,7 +1022,8 @@
       "asset-not-owned-err": "You cannot upload the asset that is not owned by you",
       "file-corrupted-err": "Your file is corrupted. Please, try to upload another one",
       "pre-issuance-details-title": "Pre-issuance info",
-      "pre-issuance-uploaded-msg": "Pre-issuance request was successfully uploaded"
+      "pre-issuance-uploaded-msg": "Pre-issuance request was successfully uploaded",
+      "learn-more-about-pre-issuance": "Learn more about pre-issuance"
```
Comment (max-nes) [inline src/i18n/en.json]: "pre-issuance-guide-link"

File: src/vue/pages/Downloads.vue [hunk 2]
```diff
@@ -59,6 +59,59 @@
...
+                  class="downloads__os-store-img"
+                  src="@static/macosx_cropped.png"
+                >
+              </a>
+            </div>
+          </div>
+          <a
+            href="https://github.com/tokend/offline-issuance"
+            class="downloads__source-code-link"
+            target="_blank">
```
Comment (max-nes) [inline src/vue/pages/Downloads.vue]: rel="noopener" here also pls

Comment (Datikzz) [reply src/vue/pages/Downloads.vue]: added

File: src/i18n/en.json (hunk 1 above)
Comment (Datikzz) [reply src/i18n/en.json]: renamed

File: src/i18n/en.json [hunk 3]
```diff
@@ -1073,7 +1077,8 @@
       "preissued-amount-lbl": "Initial pre-issued amount",
       "terms-lbl": "Upload terms",
       "terms-note": "JPG, PNG, PDF, max 5 mb.",
-      "request-submitted-msg": "Your request was successfully processed. Please, wait for approval"
+      "request-submitted-msg": "Your request was successfully processed. Please, wait for approval",
+      "learn-more-about-pre-issuance": "Learn more about pre-issuance"
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/i18n/en.json]: renamed

File: src/i18n/index.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ function buildI18n (language, ...localesJson) {
       result = _merge(
         require(`./en`),
         require(`./en.terms`),
+        require(`./en.preIssuanceGuide`),
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/i18n/index.js]: renamed

File: src/i18n/en.json [hunk 5]
```diff
@@ -1712,7 +1717,11 @@
       "mobile-apps-header": "All your assets right on your phone",
       "mobile-apps-desc": "To access your assets everywhere, get the TokenD app on your mobile device.",
       "config-header": "One important step",
-      "config-desc": "Scan this QR code with your application to provide correct configuration for TokenD environment"
+      "config-desc": "Scan this QR code with your application to provide correct configuration for TokenD environment",
+      "learn-more": "Learn more",
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/i18n/en.json]: renamed

File: src/i18n/en.json [hunk 6]
```diff
@@ -1712,7 +1717,11 @@
...
+      "pre-issuance-apps-header": "Pre-issue your tokens offline",
+      "pre-issuance-apps-desc": "To pre authorize your tokens to be issued, get the Offline Device Pre-issuance tool on your computer.",
+      "source-code": "Source code"
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/i18n/en.json]: renamed

**Pull request merged**